libkbfs/block_types.go

```diff
@@ -190,6 +225,27 @@ func (fb FileBlock) DeepCopy(codec kbfscodec.Codec) (*FileBlock, error) {
 	return &fileBlockCopy, nil
 }
 
+// UpdateHash updates the hash of this FileBlock
+func (fb *FileBlock) UpdateHash() kbfshash.RawDefaultHash {
```

Comment:
OK When I do what you ask a bunch of tests fail. I could investigate why, but I'm not sure it's worth the time sink.